src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -300,4 +325,28 @@ public void resetComments(String project, String repository, String pullRequestI
       LOGGER.debug("Exception stack trace", e);
     }
   }
+  
+  /**
+   * Get available severity to create a task.
+   */
+  public List<String> getAvailableSeverities() {
```

Comment:
why not using a switch(treshold) {
case INFO:
  result.add(INFO);
case MINOR:
...
break;
} isntead?
More readable for me